### ISSUE TITLE
cannot install it on Azure Web App or Function

### ISSUE THREAD

**Issue by franva** [scikit-fmm/scikit-fmm#52]
Python Version is 
3.7.10
Linux 

```
pip install scikit-fmm
    
    Collecting scikit-fmm
      Using cached scikit-fmm-2021.2.2.tar.gz (419 kB)
    Requirement already satisfied: numpy>=1.0.2 in /usr/local/lib/python3.7/site-packages (from scikit-fmm) (1.18.1)
    Building wheels for collected packages: scikit-fmm
      Building wheel for scikit-fmm (setup.py) ... error
      ERROR: Command errored out with exit status 1:
       command: /usr/local/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-bihhvskj/scikit-fmm_13c74930cf4a4cfc949e4a3e9
    d5f6e54/setup.py'"'"'; __file__='"'"'/tmp/pip-install-bihhvskj/scikit-fmm_13c74930cf4a4cfc949e4a3e9d5f6e54/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"',
     open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-
    5ebu19pu
           cwd: /tmp/pip-install-bihhvskj/scikit-fmm_13c74930cf4a4cfc949e4a3e9d5f6e54/
      Complete output (32 lines):
      running bdist_wheel
      running build
      running config_cc
      unifing config_cc, config, build_clib, build_ext, build commands --compiler options
      running config_fc
      unifing config_fc, config, build_clib, build_ext, build commands --fcompiler options
      running build_src
      build_src
      building extension "skfmm.cfmm" sources
      building extension "skfmm.pheap" sources
      build_src: building npy-pkg config files
      running build_py
      creating build
      creating build/lib.linux-x86_64-3.7
      creating build/lib.linux-x86_64-3.7/skfmm
      copying skfmm/pfmm.py -> build/lib.linux-x86_64-3.7/skfmm
      copying skfmm/setup.py -> build/lib.linux-x86_64-3.7/skfmm

      copying skfmm/heap.py -> build/lib.linux-x86_64-3.7/skfmm
      copying skfmm/__init__.py -> build/lib.linux-x86_64-3.7/skfmm
      running build_ext
      customize UnixCCompiler
      customize UnixCCompiler using build_ext
      customize UnixCCompiler
      customize UnixCCompiler using build_ext
      building 'skfmm.cfmm' extension
      compiling C++ sources
      C compiler: g++ -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC
    
      creating build/temp.linux-x86_64-3.7/skfmm
      compile options: '-Iskfmm -I/usr/local/lib/python3.7/site-packages/numpy/core/include -I/usr/local/include/python3.7m -c'
      g++: skfmm/fmm.cpp
      error: Command "g++ -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -Iskfmm -I/usr/local/lib/python3.7/site-packages/numpy/
    core/include -I/usr/local/include/python3.7m -c skfmm/fmm.cpp -o build/temp.linux-x86_64-3.7/skfmm/fmm.o -MMD -MF build/temp.linux-x86_64-3.7/skfmm/fmm.o.d"
     failed with exit status 127
      ----------------------------------------
      ERROR: Failed building wheel for scikit-fmm
      Running setup.py clean for scikit-fmm
    Failed to build scikit-fmm
    Installing collected packages: scikit-fmm
        Running setup.py install for scikit-fmm ... error
        ERROR: Command errored out with exit status 1:
         command: /usr/local/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-bihhvskj/scikit-fmm_13c74930cf4a4cfc949e4a3
    e9d5f6e54/setup.py'"'"'; __file__='"'"'/tmp/pip-install-bihhvskj/scikit-fmm_13c74930cf4a4cfc949e4a3e9d5f6e54/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"
    ', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-re
    cord-gku3heb3/install-record.txt --single-version-externally-managed --compile --install-headers /usr/local/include/python3.7m/scikit-fmm
             cwd: /tmp/pip-install-bihhvskj/scikit-fmm_13c74930cf4a4cfc949e4a3e9d5f6e54/
        Complete output (28 lines):
    Note: if you need to uninstall you should `pip install scikit-fmm` instead of using `setup.py install`
        running install
        running build
        running config_cc
        unifing config_cc, config, build_clib, build_ext, build commands --compiler options
        running config_fc
        unifing config_fc, config, build_clib, build_ext, build commands --fcompiler options
        running build_src
        build_src
        building extension "skfmm.cfmm" sources
        building extension "skfmm.pheap" sources
        build_src: building npy-pkg config files
        running build_py
        running build_ext
        customize UnixCCompiler
        customize UnixCCompiler using build_ext
        customize UnixCCompiler
        customize UnixCCompiler using build_ext
        building 'skfmm.cfmm' extension
        compiling C++ sources
        C compiler: g++ -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC
    
        creating build/temp.linux-x86_64-3.7/skfmm
        compile options: '-Iskfmm -I/usr/local/lib/python3.7/site-packages/numpy/core/include -I/usr/local/include/python3.7m -c'
        g++: skfmm/fmm.cpp
        error: Command "g++ -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -Iskfmm -I/usr/local/lib/python3.7/site-packages/nump
    y/core/include -I/usr/local/include/python3.7m -c skfmm/fmm.cpp -o build/temp.linux-x86_64-3.7/skfmm/fmm.o -MMD -MF build/temp.linux-x86_64-3.7/skfmm/fmm.o.
    d" failed with exit status 127
        ----------------------------------------
    ERROR: Command errored out with exit status 1: /usr/local/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-bihhvskj/s
    cikit-fmm_13c74930cf4a4cfc949e4a3e9d5f6e54/setup.py'"'"'; __file__='"'"'/tmp/pip-install-bihhvskj/scikit-fmm_13c74930cf4a4cfc949e4a3e9d5f6e54/setup.py'"'"';
    f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"
    '))' install --record /tmp/pip-record-gku3heb3/install-record.txt --single-version-externally-managed --compile --install-headers /usr/local/include/python3
    .7m/scikit-fmm Check the logs for full command output.





```

**Comment by jkfurtney** [comment]
Thanks for the message, sorry I will probably not be much help on this one. The only thing I can think is that the compiler is not installed properly in the environment you are compiling in? I think exit status 127 means the shell cannot find the specified executable (the compiler executable in this case?). If you can get a shell in that environment, you could try `python setup.py install` in the scikit-fmm source directory. When you figure it out, can you post your findings here to help others. Thanks again.

**Comment by jkfurtney** [comment]
Can you try this again, this may be fixed after the changes for #55 ?

**Comment by jkfurtney** [comment]
If someone can provide more information on this one I can have a look. Closing this issue for now.
